docs/vc-http-api.yml

```diff
@@ -143,7 +143,67 @@ components:
           "proofPurpose": "assertionMethod",
           "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..YtqjEYnFENT7fNW-COD0HAACxeuQxPKAmp4nIl8jYAu__6IH2FpSxv81w-l5PvE1og50tS9tH8WyXMlXyo45CA",
         }
-    LinkedDataProofOptions:
+    IssueCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the LinkedDataProof is created.
+      properties:
+        verificationMethod:
+          type: string
+          description: The URI of the verificationMethod used for the proof. If omitted a default assertionMethod will be used.
+        proofPurpose:
+          type: string
+          description: The purpose of the proof. If omitted "assertionMethod" will be used.
+        created:
+          type: string
+          description: The date of the proof. If omitted system time will be used.
+        challenge:
+          type: string
+          description: The challenge of the proof.
+        domain:
+          type: string
+          description: The domain of the proof.
+        credentialStatusMethod:
+          type: string
+          description: The method of credential status to issue the credential including. If omitted credential status will be included.
+      example:
+        {
+          "verificationMethod": "did:example:123#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN",
+          "proofPurpose": "assertionMethod",
+          "created": "2020-04-02T18:48:36Z",
+          "domain": "example.com",
+          "challenge": "d436f0c8-fbd9-4e48-bbb2-55fc5d0920a8",
+          "credentialStatusMethod": "RevocationList2020Status",
+        }
+    PresentCredentialOptions:
```

Comment:
Hmm, PR says it's just adding support for credential status, but many more options seem to be being added for issuance and presentation.

Reply:
See comment below really just breaking up the old shared DTO definition

Reply:
these are the json schema for options... this probably should have happened in a separate PR, but i am greatful it happened at all.

Reply:
Thanks, I missed that these were JSON Schema definitions.